src/stories/components/svg/feedback.tsx

```diff
@@ -20,7 +20,7 @@ const FeedBack = ({
       onClick={onClick}
       width={width}
       height={height}
-      viewBox='0 0 23 23'
+      viewBox={`0 0 ${width} ${height}`}
       fill='none'
```

Comment:
We can pass also the fill, that way we can use the same icon , with different color. What do you think?